src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ca/AssessmentService.java

```diff
@@ -712,6 +713,7 @@ private AssessmentSelection populateAssessmentSelectionDetails(
             asd.setRequirementValidValueCode(validValue.getKey().getValueCode());
             break;
           case INTEGER:
+            this.checkNegNumber(Integer.valueOf(c.getValue()));
```

Comment:
Think we can drop the `this` qualifier..

Reply:
Removed this